go-controller/pkg/ovn/loadbalancer.go

```diff
@@ -281,17 +312,25 @@ func (ovn *Controller) deleteLoadBalancerRejectACL(lb, vip string) {
 
 	args := []string{}
 	for _, ls := range switches {
-		args = append(args, "--", "--if-exists", "remove", "logical_switch", ls, "acl", acl)
+		args = append(args, "--", "--if-exists", "remove", "logical_switch", ls, "acl", aclUUID)
 	}
 
 	if len(args) > 0 {
 		_, _, err = util.RunOVNNbctl(args...)
 		if err != nil {
-			klog.Errorf("Error while removing ACL: %s, from switches, error: %v", acl, err)
+			klog.Errorf("Error while removing ACL: %s, from switches, error: %v", aclUUID, err)
 		} else {
-			klog.V(5).Infof("ACL: %s, removed from switches: %s", acl, switches)
+			klog.Infof("ACL: %s, removed from switches: %s", aclUUID, switches)
+		}
+	}
+
+	if len(args) > 0 {
```

Comment:
@trozet maybe I'm misreading the diff, but isn't this block redundant with the one just above at 318?

Reply:
oops copy pasta error. I'll remove that. thank.